user-interface/src/data-verification/ConsolidationCasesTable.tsx

```diff
@@ -70,15 +91,14 @@ function _ConsolidationCaseTable(
               <tr key={`${key}-case-info`} data-testid={`${key}-case-info`} className="case-info">
                 {onSelect && (
                   <td scope="row">
-                    <input
-                      type="checkbox"
+                    <Checkbox
+                      id={`${id}-case-selection-${idx}`}
```

Comment:
This might be a problem.  We should make sure that the id always starts with an alpha character.

Reply:
@btposey I know we talked a bit about the id's and data-testid's last week. Maybe we need to schedule some time to all talk and look through it?

Reply:
Yeah we should talk about it.  In any case, I moved the "id" closer to the string.